.github/workflows/e2e.gradle.workflow_dispatch.main.project-at-repo-root.slsa3.yml

```diff
@@ -0,0 +1,108 @@
+on:
+  schedule:
+    - cron: "0 15 * * *"
+  workflow_dispatch:
+    inputs:
+      trigger_build:
+        description: "internal: do not check"
+        required: false
+        default: false
+        type: boolean
+
+permissions: read-all
+
+concurrency: "e2e-gradle-workflow_dispatch-main-default-slsa3"
```

Comment:
This should be `e2e-gradle-workflow_dispatch-main-project-at-repo-root-slsa3`

Reply:
Oh, I missed that :(